Insurance_company/accounts/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.db import models
+
+User = settings.AUTH_USER_MODEL
+
+
+class Customer(models.Model):
```

Comment:
można od razu w pliku admin zarejestrować taki model, to będzie widoczny w panelu admina

Reply:
dobrze